app/models/piece.rb

```diff
@@ -10,4 +10,13 @@ def is_obstructed?(row_destination, col_destination)
     return true if is_obstructed_diagonally?(row_destination, col_destination)
     return false
   end
+
+  def capture_piece(row_destination, col_destination)
+    capture = self.game.pieces.where(row: row_destination, column:col_destination, is_black: !self.is_black)
```

Comment:
So, this will actually match pieces that are already captured, so you may want to add `captured: false` to the `where` conditions.  Alternatively, you could add a named scope to the `Piece` class that only matches active pieces:

```ruby
scope :active, -> { where(captured: false) }
scope :captured, -> { where(captured: true) }

def capture_piece(row_destination, col_destination)
  capture = self.game.pieces.active.where(row: row_destination, column:col_destination, is_black: !self.is_black)
  ...
end
```

The benefit there is you can use `pieces.active` or `pieces.captured` anywhere in your app and the naming helps people understand what you're conceptually fetching without having to parse all of the `where` conditions.